src/bentoml_cli/bentos.py

```diff
@@ -339,29 +363,60 @@ def build(  # type: ignore (not accessed)
             build_ctx=build_ctx,
         ).save(_bento_store)
 
+        containerize_cmd = f"bentoml containerize {bento.tag}"
+        push_cmd = f"bentoml push {bento.tag}"
+
         # NOTE: Don't remove the return statement here, since we will need this
         # for usage stats collection if users are opt-in.
-        if output == "tag":
-            if _usage_event_debugging():
-                # NOTE: Since we are logging all of the trackintg id to stdout
-                # We will prefix the tag with __tag__ and we can use regex to correctly
-                # get the tag from 'bentoml.bentos.build|build_bentofile'
-                click.echo(f"__tag__:{bento.tag}")
-            else:
-                # Current behaviour
-                click.echo(bento.tag)
-            return bento
-
-        if not get_quiet_mode():
-            click.echo(BENTOML_FIGLET)
-            click.secho(f"Successfully built {bento}.", fg="green")
-
-            click.secho(
-                f"\nPossible next steps:\n\n * Containerize your Bento with `bentoml containerize`:\n    $ bentoml containerize {bento.tag}",
-                fg="blue",
+        if machine:
+            click.echo(f"__tag__:{bento.tag}")
+        elif output == "tag":
+            click.echo(bento.tag)
+        elif output == "json":
```

Comment:
What's the JSON output for?

Reply:
I don't think we need json here, I was playing with a CLI use case for a prompt this weekend. I will remove it for now.